xml/System.Text/UTF32Encoding.xml

```diff
@@ -1762,9 +1762,38 @@
         <ReturnType>System.ReadOnlySpan&lt;System.Byte&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets a Unicode byte order mark encoded in UTF-32 format, if this object is configured to supply one.</summary>
+        <value>A span of bytes containing the Unicode byte order mark, if this object is configured to supply one. Otherwise, a default span.</value><remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The <xref:System.Text.UTF32Encoding> object can provide a preamble, which is an array of bytes that can be prefixed to the sequence of bytes resulting from the encoding process. Prefacing a sequence of encoded bytes with a byte order mark (code points U+0000 U+FEFF) helps the decoder determine the byte order and the transformation format, or UTF. The Unicode byte order mark (BOM) is serialized as follows (in hexadecimal):  
+  
+-   Big endian byte order: 00 00 FE FF  
```

Comment:
This would probably look better with:
```suggestion
-   Big endian byte order: `00 00 FE FF`
```

Reply:
Ok then I'll do the same on the `Preamble` property of the other classes.